### DISCUSSION TITLE
对于“生肉、不明字幕组”类别能否下载的巨大的疑问

### DISCUSSION THREAD

**Discussion by gsxgmpkt9020** [wushuo894/ani-rss#126] [Q&A]
比如这个

https://mikan.catcat.blog/Home/Bangumi/3444

![image](https://github.com/user-attachments/assets/2bb4b0d8-7665-401a-89d4-f9c3ca98263b)

六四位元字幕组被归类为“生肉、不明字幕组”，但是作为主RSS的时候根本无法预览种子文件。作为备用RSS似乎也没反应

那么
1.这是否是bug？
2.如果不是bug，那么“生肉、不明字幕组”的定位是什么？
 请帮忙测试确认下

**Comment by wushuo894** [comment]
字幕组命名不规范，无法识别，你开一下自定义集数规则

**Reply by gsxgmpkt9020** [reply]
> 自定义集数规则

1. 自定义集数规则要自己另外写正则表达式吗？还是用默认的就行了？
2. 是不是文件名里面那个神奇的五角星导致的“命名不规范”？
3. 如果开一下自定义集数规则的话，它是对于主RSS和备用RSS同时生效还是只对主RSS生效？

**Reply by gsxgmpkt9020** [reply]
> 字幕组命名不规范，无法识别，你开一下自定义集数规则

已经成功下载了，两个RSS同时生效互相补充。我来尝试理解一下：

大概就是最高优先级重载（无视其他排除规则？）把两个RSS中所有第X和X.5集全部导入到预览队列中

\d+(\.5)?：

```
\d+：

\d表示匹配一个数字字符，相当于[0-9]。
+表示前面的字符（在这里是数字）可以出现一次或多次。也就是说，\d+匹配一个或多个连续的数字。
\(和\)：

这两个符号表示一个捕获组，捕获组用于将表达式的一部分分组，以便引用或应用量词。
.：

在正则表达式中，.是一个通配符，匹配除换行符之外的任何单个字符。但是在这个表达式中，由于它在捕获组内部，所以它代表的是一个字面上的点。
\.5：

这里的\5实际上是匹配字符串中的".5"。因为.在正则表达式中是特殊字符，所以需要用反斜杠\来转义，表示字面上的点。
?：

这个问号表示前面的捕获组（即\.5）是可选的，意味着它可以出现零次或一次。
综合起来，这个正则表达式的含义是：

匹配一个或多个数字（\d+），后面可以选择性地跟随一个点和数字5（(\.5)?）。
具体来说，它可以匹配以下几种情况：
任意长度的整数，例如：1, 123, 9999。
整数后面跟随".5"的数，例如：1.5, 123.5, 9999.5。
这个正则表达式可能用于捕获带有小数点5的数值，或者只是整数
```

**Reply by wushuo894** [reply]
> 字幕组命名不规范，无法识别，你开一下自定义集数规则

1. 一般的话使用默认的就行了

2. 还是命名太奇葩

3. 同时生效，暂时不会出现冲突的情况
